File: src/test/java/test_lea_0003/a01_evenDigitsNumber/test_solution.rb
# require 'test/unit'
require 'test/unit/assertions'
include Test::Unit::Assertions
require 'test/unit/ui/console/testrunner'
require_relative '../../../../main/java/test_lea_0003/a01_evenDigitsNumber/solution.rb'


# https://dev.to/exampro/testunit-writing-test-code-in-ruby-part-1-of-3-44m2
class EvenDigitsNumberTest
    assert_equal 3, evenDigitsNumber([12, 134, 111, 1111, 10])
end

File: src/test/java/test_lea_0003/a01_evenDigitsNumber/test_solution.py
import unittest
from unittest import TestCase
from src.main.java.test_lea_0003.a01_evenDigitsNumber.solution import even_digits_number


class EvenDigitsNumberTest(TestCase):
    def test_even_digits_number(self):
        self.assertEqual(3, even_digits_number([12, 134, 111, 1111, 10]))


if __name__ == '__main__':
    unittest.main()
